apache/superset
# refactor: migrate RowCountLabel to TypeScript & added story

### SUMMARY


Non-fuctional change to migrate `superset-frontend/src/explore/components/RowCountLabel.{jsx,tsx}`.

In addiction story for `RowCountLabel` has been added. To use that storybook have been enabled for `explore` directory.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes:

![obraz](https://user-images.githubusercontent.com/3618479/150266147-6ccd2fe4-df8b-4d3c-a4d3-454dc9aeb752.png)

### TESTING INSTRUCTIONS


Use Storybook to see no visual & functional changes. You may also use build frontend.

### ADDITIONAL INFORMATION



- [x] Has associated issue: see #18100
- [x] Required feature flags: no
- [x] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351)): no

@kgabryje, cześć, czy byłbyś tak miły, aby na to spojrzeć? 😉
